spec/locale.html

```diff
@@ -0,0 +1,431 @@
+<emu-clause id="locale-objects">
+  <h1>Locale Objects</h1>
+
+  <emu-clause id="sec-intl-locale-constructor">
+    <h1>The Intl.Locale Constructor</h1>
+
+    <p>
+      The Locale constructor is a standard built-in property of the Intl object.
+    </p>
+
+    <emu-clause id="sec-apply-options-to-tag" aoid=ApplyOptionsToTag>
+      <h1>ApplyOptionsToTag( _tag_, _options_ )</h1>
+      <p>
+        The following algorithm refers to <a href="https://www.unicode.org/reports/tr35/#Identifiers">UTS 35's Unicode Language and Locale Identifiers grammar</a>.
+      </p>
+
+      <emu-alg>
+        1. Assert: Type(_tag_) is String.
+        1. If IsStructurallyValidLanguageTag(_tag_) is *false*, throw a *RangeError* exception.
+        1. Let _language_ be ? GetOption(_options_, `"language"`, `"string"`, *undefined*, *undefined*).
```

Comment:
As @spectranaut also mentioned in my PR last week, now we are stylizing strings as values on ECMA402 and ECMA262 (see #396). This way, we need to use `*"language"*` or `*"string"*`